### DISCUSSION TITLE
OSQuery client CentOS 6

### DISCUSSION THREAD

**Discussion by rdemars-zcorum** [Security-Onion-Solutions/securityonion#1937] [Unsupported Versions] [answered]
I was able to install and get running the OSQuery client rpm-launcher.rpm installed on a couple of CentOS 6 systems. The osqueryd process will run for about a 20 minutes and then stop on them. Any ideas as what it could be?

**Comment by defensivedepth** [comment, marked as answer]
I'm assuming that for those 20 minutes you can Live Query the CentOS systems? 

Unfortunately CentOS 6 is EOL on Nov 30th, 2020. As such, we have not tested on CentOS 6.  CentOS 7 is working fine from what I have seen.

I would suggest reaching out to the osquery community support Slack to confirm that CentOS 6 is still supported and see if anyone else is having the issue: https://osquery.slack.com/join/shared_invite/zt-h29zm0gk-s2DBtGUTW4CFel0f0IjTEw#/
